bedrock/firefox/templates/firefox/new/basic/thanks.html

```diff
@@ -29,6 +29,8 @@
   {% endwith %}
 {% endblock %}
 
+{% set referrals = '?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=firefox-download-thanks' %}
```

Comment:
Note for reviewer: it looks like several links on this page were 404 locally due to this missing variable. This should fix things hopefully.